crates/xilem_html/src/element/mod.rs

```diff
@@ -181,7 +182,7 @@ where
         let element = element.as_element_ref();
 
         // update attributes
-        for itm in diff_tree_maps(&prev.attributes, &self.attributes) {
```

Comment:
With everything else, I still really think this change should go away and not have a `diff` method and calling `diff_kv_iterables` here since that makes the implementation much easier to swap and experiment.

Reply:
Yeah I *mostly* agree, I have updated it (removed the extra method for the `VecMap`), makes the codesize a slightly shorter too. I've put the doc comment of the previous `VecMap::diff` method above the `diff_kv_iterables` instead.